src/common/ipfixlolib/ipfixlolib.h

```diff
@@ -125,9 +125,16 @@ See ipfixlolib.h for details on how to use this library.
 extern "C" {
 #endif
 
+/* Supported export protocols */
+typedef enum export_protocol_version {
```

Comment:
@nickbroon ?

Personally in public symbols in libraries I prefer \#defines to enums, so that there's no risk of breaking ABI.